src/orwell/game/Game.cpp

```diff
@@ -393,9 +400,10 @@ void Game::setTime(boost::posix_time::ptime const & iCurrentTime)
 
 void Game::stopIfGameIsFinished()
 {
-	if (m_gameDuration <= m_startTime - m_time)
```

Comment:
Same here: worth putting it into a dedicated function:
isGameTimeLeft() or something similar.